### PR TITLE
OEL-2166: Upgrade to BCL 1.0.0

### PR DESCRIPTION
**Pull request by brummbar**

### CODE REVIEW

File: kits/default/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "production": "npm-run-all build:*"
   },
   "devDependencies": {
-    "@openeuropa/bcl-builder": "0.24.1",
+    "@openeuropa/bcl-builder": "0.25.1",
```
Comment (drishu) [inline kits/default/package.json]: Lets use the same 28.1

File: docker-compose.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@ services:
     user: "node"
     working_dir: /home/node/app
     environment:
-      - NODE_ENV=${NODE_ENV}
+      - NODE_ENV=${NODE_ENV:-''}
```
Comment (drishu) [inline docker-compose.yml]: Can you give me a bit on info, the commit message should also be a bit more descriptive. 
`:-` evaluates to use `''` if NODE_ENV is unset or empty, but we give an empty string as default, would `-` (use '' if NODE_ENV is unset) work ?

**Review by drishu — CHANGES_REQUESTED** [review]
I also noticed inpage navigation is not working.
And while testing in showcase I noticed the offcanvas bit in the search form is visible only in mobile.

File: docker-compose.yml (hunk 2 above)
Comment (brummbar) [reply docker-compose.yml]: We'll replace with `-`  only as it's the same operation done by Docker: provides an empty string as fallback only when the variable is not defined at all.

**Comment by brummbar** [comment]
Fixed the offcanvas, for which I reviewed the updates back in the days but forgot to bring them. The inpage navigation is broken so I'll create an issue for it.

**Review by drishu — APPROVED** [review]

**Pull request merged**